docs/releases.rst

```diff
@@ -4,7 +4,57 @@
     Releases
 ======================
 
-fmf-1.4
+
+fmf-1.5.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``+`` operator now can be used for merging ``list`` of
+dictionaries with a single ``dict``. This can be for example
+useful when extending the ``discover`` step config which defines
+several phases:
+
+.. code-block:: yaml
+
+    discover:
+      - how: fmf
+        url: https://github.com/project/one
+      - how: fmf
+        url: https://github.com/project/two
+
+    /tier1:
+        summary: Run tier one tests
+        discover+:
+            filter: "tier:1"
+
+    /tier2:
+        summary: Run tier two tests
+        discover+:
+            filter: "tier:2"
+
+See the :ref:`merging<merging>` section for more details and
+examples.
+
+The ``-`` operator no more raises exception when the parent node
```

Comment:
"no longer raises exception when key is not defined by the parent node"?

Reply:
Good wording, applied in 5111c52.